spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala

```diff
@@ -162,7 +163,7 @@ class MahoutSparkILoop extends SparkILoop {
          _ __ ___   __ _| |__   ___  _   _| |_
         | '_ ` _ \ / _` | '_ \ / _ \| | | | __|
         | | | | | | (_| | | | | (_) | |_| | |_
-        |_| |_| |_|\__,_|_| |_|\___/ \__,_|\__|  version 0.12.2
+        |_| |_| |_|\__,_|_| |_|\___/ \__,_|\__|  version 0.13.0
 
```

Comment:
if it is not an expensive maintenance, i would vote to keep it. I actually occasionally use it as a scratchpad or a quick demo env